test/framework/integration_framework/iroha_instance.cpp

```diff
@@ -34,7 +34,9 @@ namespace integration_framework {
         // not required due to solo consensus
         vote_delay_(0ms),
         // amount of minutes in a day
-        mst_expiration_time_(std::chrono::minutes(24*60)),
+        mst_expiration_time_(std::chrono::minutes(24 * 60)),
+        max_rounds_delay_(3000ms),
```

Comment:
```suggestion
        max_rounds_delay_(0ms),
```
Since `IrohaInstance` is used only for testing, it should be safe to set delay to 0. Tests should be executed faster in this case.